packages/connectivity/connectivity_macos/ios/.gitignore

```diff
@@ -0,0 +1,36 @@
+.idea/
```

Comment:
Same here, most of these are already in the root `.gitignore` under `flutter/plugins`. If there's anything missing I would rather just add it there since this is going to apply to all macOS plugins.

Reply:
Done